relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -312,6 +316,12 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         }
         if let Some(country_code) = user.geo.value().and_then(|geo| geo.country_code.value()) {
             tags.insert(SpanTagKey::UserCountryCode, country_code.to_owned());
+            if let Some(subregion) = Subregion::from_iso2(&country_code.as_str()) {
+                let numerical_subregion = subregion as u8;
+                tags.insert(SpanTagKey::UserSubregion, numerical_subregion.to_string());
+            } else {
+                tags.insert(SpanTagKey::UserSubregion, "-1".to_owned()); // fallback incase of unknown region
```

Comment:
Is `-1` the best fallback here? Can we just not include the tag?

Reply:
Yeah, that's a much better idea actually! For some reason, i thought it would be convenient to query for `subregion === -1` but we can just query for `!has:subregion`